### ISSUE TITLE
Don't work with Firefox Nightly

### ISSUE THREAD

**Issue by badagoker** [downthemall/downthemall-legacy#256]
### Your DownThemAll! and Browser Version and Language and Operating System
Firefox Nightly 5701a FR Linux

### Expected Behavior
A complete browser integration

### Actual Behavior
No longer work

### Steps To Reproduce
Must be developped as a web extension

### Additional info such as log extracts
*Please enable Diagnostic Logging in the Privacy tab of the DownThemAll! preferences and either copy/paste essential part of the log or attach the entire log. Please note that the log may include sentitive information which you have to remove prior to posting it!*

**Comment by hook321** [comment]
Read this page: http://www.downthemall.net/re-downthemall-and-webextensions-or-why-why-i-am-done-with-mozilla/

**Comment by mems** [comment]
Or read this better news: http://www.downthemall.net/progress/ (it should be only a light version)

**Comment by GuardianMajor** [comment]
Anyone know what happened to the DTA website, it seems down and won't load.

**Comment by hook321** [comment]
@GuardianMajor Works fine for me.

**Comment by GuardianMajor** [comment]
It WAS down, confirmed via multiple nodes, but yes it is back up again, turns out it was probably due to a transatlantic ISP node going down, affecting people in North America and UK, thank you. I later found out my own site was not resolving from any US side DNS but only through EU, so it was pretty widespread it seems and lasted about a good 18-26 hours give or take. Seems to have resolved though.

**Comment by jtanx** [comment]
Nah, it was an expired domain, which got renewed.

**Comment by GuardianMajor** [comment]
Aha, I guess when it rains it pours ;)

**Comment by hook321** [comment]
@jtanx That was about a month ago. https://github.com/downthemall/downthemall/issues/252

**Comment by jtanx** [comment]
@hook54321a right. Well, something DNS related changed around the time it went down/came back up:

http://whois.domaintools.com/downthemall.net
(Last updated: 2017-10-03)

**Comment by hook321** [comment]
@jtanx What DNS server are you using?

**Comment by Timi7007** [comment]
Just to keep this updated: http://www.downthemall.net/delays/

**Comment by EMBBlaster** [comment]
Should be closed as duplicate of issue #240.

**Comment by wsxy162** [comment]
You may get some more updates [here](https://www.patreon.com/nmaier).
